.circleci/config.yml

```diff
@@ -16,6 +16,12 @@ jobs:
           command: |
             sudo npm install -g corepack@0.31.0
             pnpm install
+      - run:
+          name: Download GeoIP
+          command: wget "https://download.maxmind.com/app/geoip_download?edition_id=GeoIP2-City&license_key=$GEOIP_LICENSE_KEY&suffix=tar.gz" -O geoip.tar.gz
+      - run:
+          name: Untar
+          command: tar -xvzf geoip.tar.gz && mv */GeoIP2-City.mmdb geodb/
```

Comment:
How often to these files update?

Reply:
I think weekly

Reply:
less often then we deploy

Reply:
I could deploy their geoipupdate as a daemonset on all our nodes, that keeps the db up to date. And then API (and any other service) just loads from k8s host

Reply:
:hidethepain: reverting fetching code

Reply:
2859117ae9d25b59ed6e15086ece597c6c9d1ff1